common/sagas/ens.js

```diff
@@ -1,41 +1,23 @@
 // @flow
-import { takeEvery, call, put, select } from 'redux-saga/effects';
+import { takeEvery, call, apply, put, select } from 'redux-saga/effects';
 import { delay } from 'redux-saga';
-
-import { cacheEnsAddress } from 'actions/ens';
-import type { ResolveEnsNameAction } from 'actions/ens';
-
-import { getEnsAddress } from 'selectors/ens';
-import { donationAddressMap } from 'config/data';
-
+import { resolveDomainFailed, resolveDomainSuccess } from 'actions/ens';
+import type { ResolveDomainRequested, ResolveDomainFailed } from 'actions/ens';
 import type { Yield, Return, Next } from 'sagas/types';
-
-function* resolveEns(
-  action?: ResolveEnsNameAction
+import { INode } from 'libs/nodes/INode';
```

Comment:
Prefer `import type` if you only need it for typing (This should probably be a linter thing.)

Reply:
Good catch, thats a typo